include/bbp/sonata/report_reader.h

```diff
@@ -61,7 +74,8 @@ class SONATA_API SpikeReader
       private:
         Population(const std::string& filename, const std::string& populationName);
 
-        Spikes spikes_;
+        std::vector<NodeID> node_ids_;
```

Comment:
This could become a `SpikeTimes`, instead of two arrays; then the `getArrays` could be have two calling options: one without any arguments returns a `const &` to `SpikeTimes`, so no copies are required of the C++ side.